Makefile

```diff
@@ -0,0 +1,37 @@
+manage=pipenv run python manage.py
+
+dev: db.sqlite3 ## Run the local dev server under http://localhost:8000/
+	$(manage) runserver
+
+install: ## Install or update the python dependencies
+	pipenv install
+
+db.sqlite3: # if this file is not present, install everything
+	@$(MAKE) install migrate compilemessages superuser
+
+superuser: ## Create a backend user
+	$(manage) createsuperuser
+
+migrate: ## Run the database migrations
+	$(manage) migrate
+
+migrations: ## Create migrations after a model has been changed
+	$(manage) makemigrations
+
+compilemessages: ## Compile the translated messages
+	$(manage) compilemessages
+
+
+MESSAGESDIRS = participant # Space separated modules with a translation
+
+messages: $(MESSAGESDIRS) ## Update the translation files
+
+$(MESSAGESDIRS):
+		@$(MAKE) -C $@ messages
```

Comment:
Instead of putting `Makefile`s into every application of the project we could do a `cd $@ && pipenv run django-admin makemessages` here? Guess there won't be any more commands on the application level.

Reply:
You are right. 

I thought that it would be nice to `make messages` inside a specific module, but you can get exactly the same effect by doing `make participant` from the root.

I changed it.